### ISSUE TITLE
Crayons don't work

### ISSUE THREAD

**Issue by Fr-Dae** [unitystation/unitystation#8837]
Sometimes, use a pencils say "you can't use pencils on space" but it ... base's floor
tested with Pink color
and sometime some tiles don't accept ink
(tested on medbay)

**Comment by PerfectTangent** [comment]
by pencil, do you mean crayon?

**Comment by Fr-Dae** [comment]
![2022-06-16-124623_1920x1080_scrot](https://user-images.githubusercontent.com/23202421/174054151-719c20f1-ee7f-424a-a6d0-60dab500015c.png)

if you pick up a box of pencils, and use one, it says "you can't use that on space". 
you have to put the box back on a table and take it back to correct the problem.

Some tiles can't be printed

**Comment by PerfectTangent** [comment]
I was trying to confirm what you were talking about. We call them crayons. Pencils are a separate item.

**Comment by Fr-Dae** [comment]
![2022-06-16-125130_1920x1080_scrot](https://user-images.githubusercontent.com/23202421/174055067-7e15c231-df3f-43df-b986-a9ab1bdb9b72.png)

**Comment by PerfectTangent** [comment]
yeah, not an accurate translation. In english, we differentiate crayons as wax drawing tools, while pencils are graphite drawing tools. The distinction is important when both items exist in ss13 with different functionalities.

**Comment by Fr-Dae** [comment]
the coloured synthetic wooden sticks which make pretty colours, the box of which is between the plush in the psychologist's office

**Comment by Fr-Dae** [comment]
"pencil" don't exist on unitystation yet
that's why I didn't understand what you were talking about

**Comment by MaxIsJoe** [comment]
This has been fixed at some point recently.
